Ginger/GingerCore/Drivers/SeleniumDriver.cs

```diff
@@ -5441,7 +5440,25 @@ public void ActBrowserElementHandler(ActBrowserElement act)
 
                 case ActBrowserElement.eControlAction.OpenURLNewTab:
                     OpenUrlNewTab();
-                    GotoURL(act, act.GetInputParamCalculatedValue("Value"));
+
+                    if ((act.GetInputParamValue(ActBrowserElement.Fields.URLSrc) == ActBrowserElement.eURLSrc.UrlPOM.ToString()))
+                    {
+                        string POMGuid = act.GetInputParamCalculatedValue("UrlPOM");
```

Comment:
you should never use strings ad ID's, this will cause very breakable code whic compilr can't catch.
"UrlPOM" (which better be called "PomGUID") need to be added as to actBrowserAction.Fields and then refer here to the Field and not use string